### DISCUSSION TITLE
Possibility to adding `- [x]`

### DISCUSSION THREAD

**Discussion by thomasbnt** [community/community#50243] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Hi all ! 👋🏼

If possible to integrate `- [x]` to sponsor tier. 
With the preview, we can see it's work, but when we publish, we have dots and [x] not working.

_Preview sponsor tier when editing the tier_
![Preview sponsor tier when editing the tier](https://user-images.githubusercontent.com/14293805/225679072-ad223fa6-f506-42aa-b076-a7a37ac21820.png)

_Preview sponsor tier when he's published_
![Preview sponsor tier when he's published](https://user-images.githubusercontent.com/14293805/225678558-0d7d2464-c897-459f-9887-0f999a813586.png)

**Comment by Hkly** [comment, marked as answer]
Hi @thomasbnt, we have implemented this feature, thank you for your feedback!

**Reply by thomasbnt** [reply]
Oh awesome! Thanks!
